### DISCUSSION TITLE
Support for Persistent Subscriptions

### DISCUSSION THREAD

**Discussion by exosyphon-rxrevu** [yousty/event_store_client#51] [Ideas]
Hello! Curious if ya'll have Persistent Subscriptions on your roadmap. We are working in a multi-language environment with Ruby and Kotlin and would love to use Persistent Subscriptions if support was available!

https://developers.eventstore.com/clients/grpc/persistent-subscriptions.html#creating-a-subscription-group

**Comment by intale** [comment]
> Hello! Curious if ya'll have Persistent Subscriptions on your roadmap. We are working in a multi-language environment with Ruby and Kotlin and would love to use Persistent Subscriptions if support was available!
> 
> https://developers.eventstore.com/clients/grpc/persistent-subscriptions.html#creating-a-subscription-group

Hello. Yes, we planed to implement it as well, but I haven't started working under it yet.

**Reply by ncri** [reply]
@exosyphon-rxrevu We used to have it implemented in the first version of our client. But we later switched to client side subscriptions which was recommended to us by a guy from EventstoreDB, for performance reasons mainly as far as I remember, but also because it gave us control over the subscriptions. We had some issues with ES persistent subscriptions, we had failures and other issues.
